README.md

```diff
@@ -2,29 +2,55 @@
 
 [![Super-Linter](https://github.com/<OWNER>/<REPOSITORY>/actions/workflows/<WORKFLOW_FILE_NAME>/badge.svg)](https://github.com/marketplace/actions/super-linter)
 
-## Overview
+![Fluffy my Immagination](https://raw.githubusercontent.com/Alexandru-Bulai/website-project/main/fluffy-my-pillow.gif)
 
-This project involves creating a responsive website for Pet Care company that features a modern design with a navigation menu, company logo, title header, and a central GIF. The website is built using HTML, Tailwind css and JavaScript using Jest as testing enviroment.
+
+## Project Overview
+
+This project aims to develop a comprehensive and responsive website for a Pet Care company, showcasing its services and commitment to pet well-being through a visually appealing and user-friendly interface. The website is meticulously crafted using the latest web technologies to ensure high performance and accessibility across various devices.
+
+## Key Features
+
+1. **Central GIF Animation**: A captivating central GIF animation that engages visitors, showcasing the lively and caring nature of the Pet Care company's services.
+2. **Interactive Elements**: Utilizes JavaScript to add interactivity to the website, including dynamic content loading and interactive UI components.
+3. **Tailwind CSS**: Employs Tailwind CSS for styling, taking advantage of its utility-first approach to build custom designs quickly without leaving the HTML.
+4. **Accessibility and SEO**: Implements best practices for accessibility(`SEO score: 77`) and SEO (`SEO score: 92`), ensuring the website is accessible to as wide an audience as possible and ranks well on search engines.
```

Comment:
Seo twice? Also where are the numbers coming from? A link would be better. If not an image should be ok as well

Reply:
i runned the test locally can provide link im afraid